doc/travis_ci.md

```diff
@@ -1,11 +1,32 @@
 # Travis CI
 
-For many use cases, the
-[Travis caching section of the user guide](GUIDE.md#travis-with-caching)
-will be sufficient.
-
-This page documents how to use Stack on [Travis CI](https://travis-ci.org/). We
-assume you have basic familiarity with Travis.
+This page documents how to use Stack on
+[Travis CI](https://travis-ci.org/). We assume you have basic
+familiarity with Travis. We provide two fully baked example files
+ready to be used on your projects:
+
+* [The simple Travis configuration](https://raw.githubusercontent.com/commercialhaskell/stack/master/doc/travis-simple.yml)
+  is intended for applications that do not require multiple GHC
+  support or cross-platform support. It builds and tests your project
+  with just the settings present in your `stack.yaml` file.
+* [The complex Travis configuration](https://raw.githubusercontent.com/commercialhaskell/stack/master/doc/travis-complex.yml)
+  is intended for projects that need to support multiple GHC versions
+  and multiple OSes, such as open source libraries to be released to
+  Hackage. It tests against cabal-install, as well as Stack on Linux
+  and OS X. The configuration is significantly more involved to allow
+  for all of this branching behavior.
+
+Each of these configurations is ready to be used immediately, just
+copy-paste the content into the `.travis.yml` file in the root or your
+repo, enable Travis on the repo, and you're good to go.
```

Comment:
I'd point out that the latter configuration might often need customization. It makes choices on which GHC versions to support, and assumes that the same `stack.yaml` (including extra-deps) works on all the chosen resolvers—that's likely if no extra-deps are used. Maybe add a pointer to the relevant sections below?